etl/moped_projects/queries.py

```diff
@@ -0,0 +1,43 @@
+# Gets the ID and location of crashes
+query_vz = """
+    {
+      crashes(
+        where: {
+          position: { _is_null: false },
+          is_deleted: { _eq: false },
+          is_temp_record: { _eq: false }
+        }
+      ) {
+        id,
+        position
+      }
+    }
+"""
+
+# Gets the ID and geometry for every record in the component_arcgis_online_view table
+query_moped = """
+    {
+        component_arcgis_online_view(where: { geometry: { _is_null: false } }) {
+            project_component_id,
```

Comment:
Commas not creating a syntax error in GraphQL is a TIL for me. Cool to know! https://spec.graphql.org/June2018/#sec-Insignificant-Commas